src/node/rpc/test/frontend_test.cpp

```diff
@@ -508,26 +508,6 @@ TEST_CASE("process")
     CHECK(value.req.empty());
     CHECK(value.sig == signed_call[jsonrpc::SIG]);
   }
-
-  // TODO: verify_client_signature in HTTP
```

Comment:
Has this TODO been implemented or ticketed?

Reply:
I think it is, and looking at frontend.h, it seems to be, but I can see that some of the perf tests aren't enabled (sig forwarding for example), so I'll double check what's up there.

Reply:
So the check is implemented, but it fails with the C++ test client for some reason, and that's ticketed here: https://github.com/microsoft/CCF/issues/787